index.html

```diff
@@ -17,5 +17,61 @@
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+    <nav class="navbar">
```

Comment:
Some of the code in here is a little messy and could be indented better. Fortunately Visual Studio Code can do this for you.

You can find out here here:
https://stackoverflow.com/a/29973358